Process_guide.md

```diff
@@ -123,7 +123,7 @@ KBase is complex enough that per-repository tests will never be sufficient to un
 
 ci.kbase.us will provide an environment for conducting integration tests. This system will update regularly against defined branches for each repo and will run integration tests defined a special github repo. Developers should add to this integration suite in the same way they do for unit-tests.
 
-We have three primary deployment environments. The first is -ci, which generally tracks the develop branch and is deployed frequently. Automated integration testing will happen here, and the bar for passing and being promoted to the next stage will be high. The second stage is -next, which is intended to be as close to what will be deployed to production as possible, and is (currently) deployed weekly. When a service has been deployed in -next for long enough (~1-2 weeks, though this is flexible) the author can request that it be deployed to the production environment. ([*This doc*](https://docs.google.com/a/lbl.gov/document/d/1SwUUMHYPjnM77utZM5WSE7uatpVsdJo78U5pJYmxtUU) will describe the deploy environments in more detail.)
+We have three primary deployment environments. The first is -ci, which generally tracks the develop branch and is deployed frequently. Automated integration testing will happen here, and the bar for passing and being promoted to the next stage will be high. The second stage is -next, which is intended to be as close to what will be deployed to production as possible, and is (currently) deployed weekly. When a service has been deployed in -next for long enough (~1-2 weeks, though this is flexible) the author can request that it be deployed to the production environment. ([*This doc*](https://github.com/kbase/project_guides/blob/master/Developer_deployment.md) will describe the deploy environments in more detail.)
```

Comment:
You should use a local URL instead of an absolute one to github. We want the local URLs to render correctly later on, not to depend to the GH location.

Otherwise looks good!